rotkehlchen/globaldb/schema.py

```diff
@@ -70,6 +70,10 @@
 INSERT OR IGNORE INTO asset_types(type, seq) VALUES ('V', 22);
 /* OTHER */
 INSERT OR IGNORE INTO asset_types(type, seq) VALUES ('W', 23);
+/* AVALANCHE TOKEN */
+INSERT OR IGNORE INTO asset_types(type, seq) VALUES ('X', 24);
+/* SOLANA TOKEN */
+INSERT OR IGNORE INTO asset_types(type, seq) VALUES ('Y', 23);
```

Comment:
Hmm I think your problem is here :sweat: 

```suggestion
INSERT OR IGNORE INTO asset_types(type, seq) VALUES ('Y', 25);
```